test/functional/lib/tests.cpp

```diff
@@ -3133,8 +3134,9 @@ GenericRxMatch(
                MatchType == XDP_MATCH_IPV6_UDP_PORT_SET ||
                MatchType == XDP_MATCH_IPV4_TCP_PORT_SET ||
                MatchType == XDP_MATCH_IPV6_TCP_PORT_SET) {
-        PortSet.reset((UINT8 *)calloc(XDP_PORT_SET_BUFFER_SIZE, 1));
+        PortSet.reset((UINT8 *)AllocMem(XDP_PORT_SET_BUFFER_SIZE));
```

Comment:
please create a wrapper for calloc that zeros, or let allocmem zero memory by default